openfecwebapp/config.py

```diff
@@ -28,7 +28,7 @@
 )
 
 features = {
-    'legal': bool(env.get_credential('FEC_FEATURE_LEGAL', '')),
+    'legal_statutes': bool(env.get_credential('FEC_FEATURE_LEGAL_STATUTES', '')),
```

Comment:
@porta-antiporta mentioned in stand-up that we should leave the code as-is for now, with the new feature flag.